GPU/GLES/FramebufferManagerGLES.cpp

```diff
@@ -845,6 +845,21 @@ static void LogReadPixelsError(GLenum error) {
 
 static void SafeGLReadPixels(GLint x, GLint y, GLsizei w, GLsizei h, GLenum fmt, GLenum type, void *pixels) {
 	CHECK_GL_ERROR_IF_DEBUG();
+
+	// Apply the correct alignment.
+	if (fmt == GL_RGBA && type == GL_UNSIGNED_BYTE) {
+		glPixelStorei(GL_PACK_ALIGNMENT, 4);
+	} else if (fmt == GL_RGB && type == GL_UNSIGNED_BYTE) {
+		glPixelStorei(GL_PACK_ALIGNMENT, 1);
+	} else if (fmt == GL_DEPTH_COMPONENT && type == GL_FLOAT) {
+		glPixelStorei(GL_PACK_ALIGNMENT, 4);
+	} else if (fmt == GL_STENCIL_INDEX && type == GL_UNSIGNED_BYTE) {
+		glPixelStorei(GL_PACK_ALIGNMENT, 2);  // This seems dubious. 1?
```

Comment:
Hmm, yes, looks like a typo.  In practice, that read was always a full framebuffer, so always aligned to 16 pixels and reading into a buffer that's probably aligned to 16 or at least 8.  So maybe we just never noticed.

-[Unknown]